wikipedia-sentences-incremental/app.py

```diff
@@ -0,0 +1,112 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import itertools as it
+
+import click
+from jina.flow import Flow
+from jina import Document
+
+
+def config():
+    os.environ['JINA_DATA_FILE_1'] = os.environ.get(
+        'JINA_DATA_FILE_1', 'data/input-1.txt'
+    )
+    os.environ['JINA_DATA_FILE_2'] = os.environ.get(
+        'JINA_DATA_FILE_2', 'data/input-2.txt'
+    )
+    os.environ["JINA_WORKSPACE"] = os.environ.get("JINA_WORKSPACE", "workspace")
+
+    os.environ["JINA_PORT"] = os.environ.get("JINA_PORT", str(45678))
+    # os.environ["JINA_MAX_DOCS"] = os.environ.get("JINA_MAX_DOCS", str(100))
+
+
+def print_topk(resp, sentence):
+    for d in resp.search.docs:
+        print(f"Ta-Dah🔮, here are what we found for: {sentence}")
+        for idx, match in enumerate(d.matches):
+
+            score = match.score.value
+            if score < 0.0:
+                continue
+            print(f'> {idx:>2d}({score:.2f}). {match.text}')
+
+
+def index(num_docs):
+    f = Flow().load_config("flows/index.yml")
+
+    with f:
+        print(f"Indexing {os.environ['JINA_DATA_FILE_1']}")
+        data_path = os.path.join(os.path.dirname(__file__), os.environ.get('JINA_DATA_FILE_1', None))
+        f.index_lines(filepath=data_path, batch_size=16, read_mode='r', size=num_docs)
+
+    with f:
+        print(f"Indexing {os.environ['JINA_DATA_FILE_2']}")
+        data_path = os.path.join(os.path.dirname(__file__), os.environ.get('JINA_DATA_FILE_2', None))
+        f.index_lines(filepath=data_path, batch_size=16, read_mode='r', size=num_docs)
```

Comment:
We use `request_size` instead of `batch_size` now

Reply:
Done